composer.json

```diff
@@ -17,7 +17,7 @@
     "require": {
         "php": ">=8.0",
         "ext-pdo": "*",
-        "utopia-php/database": "0.20.0"
+        "utopia-php/database": "dev-datetime-tz-filter as 0.20.0"
```

Comment:
reminder to revert this to 0.21.0 when its release

Reply:
Yep!